demo/Snakefile

```diff
@@ -54,6 +55,7 @@ config["lcr-modules"]["_shared"]["samples"] = SAMPLES
 
 
 # Load module-specific snakefiles
+include: "../modules/gridss/1.0/gridss.smk"
 include: "../modules/salmon/1.0/salmon.smk"
 include: "../modules/bam2fastq/1.0/bam2fastq.smk"
```

Comment:
Oh, weird. It's still 1.0 on master. Why don't we wait to update it until Kostia's fixes are approved?

Reply:
Kostia's changes should probably still be called version 1.1 because the version 1.1 in master doesn't really work. I don't know if other things will need to change outside the snakefile for the patched version to work.